extension/package.json

```diff
@@ -61,6 +61,19 @@
 							"Unix connects to an X11 server listening on a unix socket specified in `#remoteX11.X11Socket#`. This is generally nessasary for linux hosts."
 						]
 					},
+					"remoteX11.XAuthPermissionLevel": {
```

Comment:
I've moved the settting. In theory the authentication level is agnostic to the connection method even if only used by the SSH extention.

Reply:
> In theory the authentication level is agnostic to the connection method even if only used by the SSH extention.

True, though since nothing but SSH uses it I think it makes more sense for it to belong to the SSH extension, at least for now.